### DISCUSSION TITLE
The last few releases do not include Eclipse babel localization

### DISCUSSION THREAD

**Discussion by bianyun** [dbeaver/dbeaver#20649] [DBeaver Development] [answered]
I found that the last few releases (including the latest release: 23.1.3) do not include Eclipse babel localization, this resulted in some menus not being localized. But I remember that the earlier versions would include the localization of Eclipse babel before publishing.

Currently I have to manually install Eclipse babel localizations after each update. So was this a mistake or was it intentional?

**Comment by serge-rider** [comment, marked as answer]
Yes, thanks for noticing. 
Something went wrong during release packaging. Will be fixed in the next release.
